staging/docker-registry/values.yaml

```diff
@@ -272,8 +272,8 @@ statefulSet:
   syncer:
     enabled: true
     image:
-      repository: regclient/regsync
-      tag: latest
```

Comment:
should these be applied as a patch on top of upstream values?

Reply:
Ahh good point, fixing.

Reply:
These are new values added in the fork, added another patch so we can easily update the regsync version in the future.